pyproject.toml

```diff
@@ -112,7 +112,17 @@ include = [
 ]
 
 [tool.pdm]
-ignore_package_warnings = ["sphinx", "slotscheck"]
+ignore_package_warnings = [
+  "alabaster",
+  "sphinxcontrib-*",
+  "sphinx-*",
+  "sphinx",
+  "pre-commit",
+  "pydata-sphinx-*",
+  "slotscheck",
+  "autobahn",
+  "accessible-pygments"
+]
```

Comment:
What command do you run that produces these warnings?

Reply:
just the normal `make install`.  I get a warning about these libraries not having 3.8 support.